extensions/amp-story/0.1/amp-story.js

```diff
@@ -91,6 +91,19 @@ const DESKTOP_HEIGHT_THRESHOLD = 550;
 /** @private @const {number} */
 const MIN_SWIPE_FOR_HINT_OVERLAY_PX = 50;
 
+/**
+ * The duration of time (in milliseconds) to wait for a page to be loaded.
```

Comment:
`...for a  page to be loaded before story becomes visible.`

Reply:
Done.